repositories/README.md

```diff
@@ -0,0 +1,5 @@
+All repository data is imported via the "images" library, see
+https://github.com/osbuild/images/tree/main/data/repositories
+
+Most are imported via the "vendor" mechanism, but some special
+repositories are kept here.
```

Comment:
:thinking: Maybe this should also mention that we keep the symlinks here as go:embed does not support those